hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-1.x-replication/pom.xml

```diff
@@ -0,0 +1,175 @@
+
+  <groupId>com.google.cloud.bigtable</groupId>
+  <artifactId>bigtable-hbase-1.x-replication</artifactId>
+  <version>0.0.1-SNAPSHOT</version> <!-- {x-version-update:bigtable-hbase-replication:current} -->
+
+  <packaging>jar</packaging>
+  <name>bigtable-hbase-1.x-replication</name>
+  <description>Library to enable one way replication from HBase to Cloud
+    Bigtable.
+  </description>
+
+  <properties>
+    <!-- define a property that can be ignored by renovate -->
+    <hbase1-hadoop-slf4j.version>1.6.1</hbase1-hadoop-slf4j.version>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <!-- disable google-cloud-shared-config enforcement checks because
+          hbase-client's dependency subtree doesn't follow the same rules and is
+           unable to be fixed here -->
+          <execution>
+            <id>enforce</id>
+            <configuration>
+              <skip>true</skip>
+            </configuration>
+          </execution>
+          <!-- This must be a drop in replacement for hbase-client, so can't
+            manage hbase's deps -->
+          <execution>
+            <id>enforce-banned-deps</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <skip>true</skip>
+            </configuration>
+          </execution>
+          <execution>
+            <id>enforce-java</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireJavaVersion>
+                  <version>[1.8,)</version>
+                </requireJavaVersion>
+              </rules>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
+      </plugin>
+      <plugin>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <version>3.3.0</version>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>single</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <descriptorRefs>
+            <descriptorRef>jar-with-dependencies</descriptorRef>
+          </descriptorRefs>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+  <dependencies>
+    <!-- bigtable libs for org.apache.hadoop.hbase -->
+    <dependency>
+      <groupId>com.google.cloud.bigtable</groupId>
+      <artifactId>bigtable-hbase-replication-core</artifactId>
+      <version>0.0.1-SNAPSHOT
+      </version> <!-- {x-version-update:bigtable-hbase-replication:current} -->
```

Comment:
please make sure to keep the comment on the same line as the version

Reply:
done. Checked in all the new POMs.

Reply:
I double checked all the new POMs. all good.